app/src/main/java/com/rockthevote/grommet/ui/registration/UnknownDataSource.kt

```diff
@@ -0,0 +1,19 @@
+package com.rockthevote.grommet.ui.registration
+
+import com.rockthevote.grommet.data.db.model.GeoLocation
+
+/**
+ * TODO: I don't know where this data is derived from?
+ *
+ * DO NOT MERGE THIS, it is a placeholder while I figure out where data comes from
+ * for the API
+ */
+data class UnknownDataSource(
+    val partnerOptInEmail: Boolean,
+    val partnerOptInSms: Boolean,
+    val partnerOptInVolunteer: Boolean,
+    val sourceTrackingId: String,
+    val partnerTrackingId: String,
+    val geoLocation: GeoLocation,
+    val openTrackingId: String
```

Comment:
name of the partner company doing it (i.e. the canvassing org the canvasser works for)